src/routes/userRoutes.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const { getUser, signupUser, loginUser, logoffUser, getAllUsers, resetPassword, updateUserProfile } = require('../controllers/userController');
+const { getUser, signupUser, loginUser, logoffUser, getAllUsers, resetPassword, updateUserProfile, deleteUser, deleteAllUsers } = require('../controllers/userController');
```

Comment:
I don't see `deleteUser` and `deleteAllUsers` added in this commit. Are these real functions? Why weren't they imported in previous MRs and are they used anywhere in this file to justify importing if so?

Reply:
I was thinking of adding deleteUser and deleteAllUsers functions. I will add these two functions. For the front end, we don’t need these functions; they are just for the backend team during testing.